src/lib/helpers/validateURL.ts

```diff
@@ -13,8 +13,9 @@ export async function isURLAllowed(url: string): Promise<boolean> {
   } catch (err: any) {
     if (!VALIDATE_URL_IGNORED_ERRORS.some((msg) => err.message.includes(msg))) {
```

Comment:
Looking at `validateURL` code I don't see how it can throw `ENOTFOUND` or `EAI_AGAIN` (`VALIDATE_URL_IGNORED_ERRORS`), maybe it got outdated, so I guess this if will always be `true` and block the url

Reply:
it doest not throw those specific errors, but it encapsulates them in `DNSResolveError `
cc `[FETCH][DNS] Resolving error: ${originalError.message}`